crates/rust-analyzer/src/cli/load_cargo.rs

```diff
@@ -33,7 +33,7 @@ pub fn load_workspace_at(
     load_workspace(workspace, load_config, progress)
 }
 
-pub fn load_workspace(
+fn load_workspace(
```

Comment:
The method is used in the load_workspace_at only and that one is used in benchmarks, analysis stats, cli diagnostics and ssr.

To me it looks fine to enable priming unconditionally, but maybe it's better to add a boolean parameter instead?

Reply:
Switched to populating them via parameter, currently populated only for completion benchmark now.